mungegithub/mungers/lgtm_handler.go

```diff
@@ -83,7 +98,46 @@ func (LGTMAfterCommitMunger) Munge(obj *github.MungeObject) {
 	}
 }
 
-func (LGTMAfterCommitMunger) isStaleComment(obj *github.MungeObject, comment *githubapi.IssueComment) bool {
+func hasValidReviwer(reviewers []*githubapi.User) bool {
+	for _, r := range reviewers {
+		if r != nil && r.Login != nil {
+			return true
+		}
+	}
+	return false
+}
+
+func foundLGTMFromReviewer(comments []*githubapi.IssueComment, reviewers []*githubapi.User, lastModified *time.Time) bool {
+	for _, c := range comments {
+		if lastModified == nil || c.CreatedAt == nil || (*lastModified).After(*c.CreatedAt) {
+			continue
+		}
+		if !isReviewer(c.User, reviewers) {
```

Comment:
What I think @bgrant0607 meant is that: if somebody that is NOT the reviewer says "LGTM", it would be nice to mention the reviewer, rather than just ignore